extensions/amp-cxense/0.1/amp-cxense.js

```diff
@@ -0,0 +1,276 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {CSS} from '../../../build/amp-cxense-0.1.css';
+import {getLengthNumeral, isLayoutSizeDefined} from '../../../src/layout';
+import {loadPromise} from '../../../src/event-helper';
+import {setStyles} from '../../../src/style';
+import {timer} from '../../../src/timer';
+
+const DEFAULT_AMD_SRC = location.protocol + '//embed.widget.cx/amd.js';
+const DEFAULT_EMBED_APP = '/app/player/m4/dist/';
+const PRE_CONNECT_URLS = [DEFAULT_AMD_SRC].concat([
+    location.protocol + '//api.widget.cx',
+    location.protocol + '//api.ramp.com' // todo: rm when safe
+]);
+const DATA_ATTR_NAME_MIDFIX = '-ramp-'; // todo: change to -cx- when safe
+
+let skipTransfer = {
+    'data-isplayer': true,
+    'data-noui': true,
+    'data-placeholder': true,
+    'class': true,
+    'layout': true
+};
+
+let _getLongDataAttributeName = (name) => {
```

Comment:
Same thing with methods, suffix `_` for private. 

Also please add annotation for this function.

```
/**
 * @param {string} name
 * @return {string}
 */
```